msteams-platform/concepts/build-and-test/deep-links.md

```diff
@@ -337,6 +337,20 @@ The query parameters are:
 > [!NOTE]
 > You can see `channelId` and `groupId` in the URL from the channel.
 
+### Generate deep links to chat messages
+
+Use this deep link format to navigate to a message in individual and group chats in Teams:
+
+`http://teams.microsoft.com/l/message/{chatId}/{messageId}?context={contextObject}`
+
+Example: `http://dev.local/l/message/19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces/1563480968434?context={"contextType"%3A"chat"}`
+
+The query parameters are:
+
+* `chatId`: ChatId that is available from the tab context for group and meeting chat. For example, `19:253f5895-9a62-4362-8d38-43f0205c702c_f1b94dcf-0aa3-4989-bcdf-ef4a5ed00f86@unq.gbl.spaces`.
```

Comment:
We should say that apps can read a group chat id through context (JS SDK) or from the incoming request payload to Bot.